module-bsp/drivers/i2c/DriverI2C.hpp

```diff
@@ -49,6 +51,8 @@ namespace drivers
 
         virtual ssize_t Modify(const I2CAddress &addr, const uint32_t mask, bool setClr, const size_t size) = 0;
 
+        virtual status_t GetLastError() = 0;
```

Comment:
This may be non-virtual as well, because returning the last error probably won't have to be a customizable operation